setup.py

```diff
@@ -61,6 +61,7 @@
         'eth-utils>=1.0.2,<2',
         'ipfsapi>=0.4.3,<1',
         'jsonschema>=2.6.0,<3',
+        "protobuf>=3.0.0",
```

Comment:
Can you put an upper bound on this so we don't accidentally do a breaking upgrade.